changelog.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
-## [1.0.0] - [Unreleased]
+## [1.0.1] - [UNRELEASED]
+
+### Added
+1. Functionality to authenticate against a protected database.
```

Comment:
If functionality, version should change to `1.1.0` as per semver else move it under bug fix.